### DISCUSSION TITLE
How to control the vehicle using known speed and continuous trajectory coordinates？

### DISCUSSION THREAD

**Discussion by CHDWeiCheng** [carla-simulator/carla#4458] [Q&A]
Hello，I have used deep reinforcement learning to predict the future trajectory and speed of a vehicle, and I want to apply the results to CARLA. But I don't know how to use the continuous velocity and trajectory coordinates to control a vehicle to do the corresponding action, which python API should I use? Also, I want to export the future trajectory to the animation of CARLA to form a dynamic future trajectory line in front of the vehicle, how should I do？

**Comment by SExpert12** [comment]
Hi, are you able to find out solution of your question? Please share it because I am also working on the same topic.
